jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java

```diff
@@ -881,11 +881,27 @@ private TTransport createUnderlyingTransport() throws TTransportException {
       }
     } else {
       // get non-SSL socket transport
-      transport = HiveAuthUtils.getSocketTransport(host, port, loginTimeout);
+      transport = HiveAuthUtils.getSocketTransport(host, port, loginTimeout, maxMessageSize);
     }
     return transport;
```

Comment:
nit: can we set the `maxMessageSize` by `transport.getConfiguration().setMaxMessageSize()` for both non-SSL and SSL transport?

Reply:
That is a great idea. I am not sure how I missed that approach - likely in a rush to fix the issue.

Reply:
I went with implementing this solution within the calls HiveAuthUtils calls so that it is handled uniformly.